src/OFS/Image.py

```diff
@@ -878,12 +875,6 @@ class Image(File):
     security.declareProtected(View, 'get_size')  # NOQA: D001
     security.declareProtected(View, 'getContentType')  # NOQA: D001
 
-    if bbb.HAS_ZSERVER:
-        security.declareProtected(change_images_and_files, 'PUT')  # NOQA: D001
```

Comment:
This one should maybe remain?

Reply:
It is already declared in the base class `File`. See https://github.com/zopefoundation/Zope/blob/06e2b82d7ee6421baea95853158708a2f6ed910a/src/OFS/Image.py#L692-L694